core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -121,6 +122,15 @@
  *
  */
 public class InodeSyncStream {
+  /**
+   * Return status of a sync result.
+   */
+  public enum SyncStatus {
```

Comment:
I don't understand how this is in `InodeSyncStream`, but this file does not use this enum.

Reply:
agree. as proposed above, we can make `#sync` to return this value